chalice/tests/chalicelib/criteria/test_aws_support_rds_security_groups.py

```diff
@@ -76,3 +115,27 @@ def setUp(self):
         """
         super(TestAwsSupportRDSSecurityGroupsRed, self).setUpClass()
         self.subclass = AwsSupportRDSSecurityGroupsRed(self.app)
+
+    def test_evaluate_pass(self):
+        """
+        Compliant case
+        """
+        # input params
+        event = {}
+        whitelist = []
+        for item in self.test_data["pass"]["result"]["flaggedResources"]:
+            # tests
+            output = self._evaluate_invariant_assertions(event, item, whitelist)
+            self._evaluate_passed_status_assertions(item, output)
+
+    def test_evaluate_fail(self):
+        """
+        Compliant case
```

Comment:
are both the pass and fail supposed to have the same docstring? I am not sure they need any docstring really.